react/src/components/AssetDetail/AssetDetail.tsx

```diff
@@ -23,37 +30,39 @@ const AssetDetail: React.FC<AssetModalProps> = ({
   const featureSource: string =
     geoapiUrl + '/assets/' + selectedFeature?.assets?.[0]?.path;
 
-  const fileType = getFeatureType(selectedFeature);
+  const fileType: FeatureType = getFeatureType(selectedFeature);
```

Comment:
```suggestion
  const featureType: FeatureType = getFeatureType(selectedFeature);
```
i would switch to the variable name to be clear; same suggestion applies to AssetButton